pkg/skaffold/runner/runner.go

```diff
@@ -428,7 +428,10 @@ func dependenciesForArtifact(a *latest.Artifact) ([]string, error) {
 
 	var p []string
 	for _, path := range paths {
-		p = append(p, filepath.Join(a.Workspace, path))
+		if !filepath.IsAbs(path) {
```

Comment:
This should be fine in regards to the other artifacts right? There should be no case where a dep would be absolute yet need to resolve against workspace?